pkg/crosscluster/streamclient/pgconn.go

```diff
@@ -107,6 +107,9 @@ func uriWithInlineTLSCertsRemoved(remote url.URL) (url.URL, *tlsCerts, error) {
 		// are deprecated in the stdlib. For now, I've skipped
 		// it.
 		block, _ := pem.Decode([]byte(key))
+		if block == nil {
```

Comment:
We should validate we can parse the cert when the user first tries to create the external connection. Having to debug a failing job is much more annoying than seeing an create external connection failure.

Reply:
Yeah I agree. The `nil` checks here should still be done though just for completeness and avoid panics instead of relying on an assumption that the query parameters have already been validated